source/documentation/runbooks/dns/ns-changes-involving-gds.html.md.erb

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Changes to Nameserver Records involving GDS Domains Team
+last_reviewed_on: 2023-05-22
+review_in: 3 months
+---
+
+# Changes to Nameserver Records involving GDS Domains Team
+
+This runbook is created to help with planning and implementation of Nameserver record changes that will involve the GDS Domains Team. This would normally occur when we are migrating a service from one hosting platform to another.
+
+This will typically be service that have domain names in the format `example.service.gov.uk`, `example.gov.uk` or `example.independant-inquiry.gov.uk`.
+
+## Check if this change will involve BT or direct engagement with the GDS Domains Team.
+
+There are two slightly different processes depending on where we manage our domains. If the domain is present in BT-DNS then this process will involve BT, otherwise change requests will need to be made to the GDS Domains Team.
```

Comment:
How do I know if it's present in "BT-DNS"?

Reply:
Added link to BT-DNS portal